exporters/stdout/stdoutmetric/config.go

```diff
@@ -74,6 +81,21 @@ func WithEncoder(encoder Encoder) Option {
 	})
 }
 
+// WithWriter sets the export stream destination.
+// If `WithEncoder` is also used, this option will have no effect.
+func WithWriter(w io.Writer) Option {
+	return WithEncoder(json.NewEncoder(w))
+}
+
+// WithPrettyPrint sets the export stream format to use JSON.
+// If `WithEncoder` is also used, this option will have no effect.
```

Comment:
This comment is incorrect. I would say that this function works when the exporter is configured to use a `[json.Encoder]` encoder.